Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -57,6 +57,39 @@ open class ChatChannelListVC: _ViewController,
 
     /// Used for mapping `ListChanges` to sets of `IndexPath` and verifying possible conflicts
     private let collectionUpdatesMapper = CollectionUpdatesMapper()
+    
+    /// Create a new ChatChannelListViewController
```

Comment:
I think the types in the method docs should be between ``, e.q. `ChatChannelListViewController`.

Reply:
Yep great spot @martinmitrevski